packages/xs-extra/wsproxy.master/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+name: "wsproxy"
+version: "1.1.0"
+authors: [ "Jon Ludlam" "jonathan.ludlam@citrix.com"
+           "Marcello Seri <marcello.seri@citrix.com>" ]
+license: "LGPL-2.0 with OCaml linking exception"
+homepage: "https://github.com/xapi-project/wsproxy"
+dev-repo: "https://github.com/xapi-project/wsproxy"
+bug-reports: "https://github.com/xapi-project/wsproxy/issues"
+build: [
+  ["oasis" "setup"]
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: [
+  ["ocaml" "setup.ml" "-install"]
+]
+remove: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-uninstall"]
+]
+depends: [
+  "lwt" {build}
```

Comment:
@mseri PR to fix the `opam` file in the `vncproxy` repo: https://github.com/xapi-project/vncproxy/pull/5